### DISCUSSION TITLE
Update to German locale [de]

### DISCUSSION THREAD

**Discussion by c-wolke** [aclap-dev/video-downloadhelper#913] [Language & Translation]
[de.txt](https://github.com/user-attachments/files/15942893/de.txt)

**Comment by Floskelkoenig** [comment]
[de.txt](https://github.com/user-attachments/files/15950679/de.txt)
Translation for 9 lines

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
